go.mod

```diff
@@ -1,25 +1,33 @@
 module github.com/zitadel/zitadel-go/v2
 
-go 1.19
+go 1.22.4
```

Comment:
Note we support the last 2 releases of Go. Therefore the go version for the module cannot be higher than 1.21.

Reply:
Thanks for your review @muhlemmer 
To confirm, should we wait for the next branch to incorporate these changes and close this current merge request?

Reply:
better close this and I you want you can open a PR to the `next` branch. Otherwise we'll take care of it.